libr/include/r_vector.h

```diff
@@ -74,6 +74,11 @@ static inline bool r_vector_empty(const RVector *vec) {
 	return vec->len == 0;
 }
 
+// backward compat
+R_DEPRECATE static inline void r_vector_clear(RVector *vec) {
```

Comment:
I don't agree on this "deprecate" thing. "clear" is a very common method name for vectors, more than "fini" (which to me is actually confusing). So I would keep this, without the R_DEPRECATE. @thestr4ng3r ?

Reply:
agreed